zerver/openapi/zulip.yaml

```diff
@@ -10641,6 +10652,36 @@ paths:
                           ],
                         "result": "success",
                       }
+    patch:
+      operationId: reorder-linkifiers
+      summary: Reorder all linkifiers
+      tags: ["server_and_organizations"]
+      description: |
+        Change the order that [linkifiers](/help/add-a-custom-linkifier)'s
+        regular expression patterns are matched in messages and topics.
+        Useful for defining linkifiers with overlapping patterns.
+
+        **Changes**: New in Zulip 8.0 (feature level 202). Before that,
+        linkifiers were always processed in order by ID, which meant users would
+        need to delete and recreate them in order to modify the processing
+        order.
```

Comment:
Revised text here to prevent posssesive with plural format, as well as some other clarifications ...

```markdown
        Change the order that the regular expression patterns in the organization's
        [linkifiers](/help/add-a-custom-linkifier) are matched in messages and topics.
        Useful when defining linkifiers with overlapping patterns.

        **Changes**: New in Zulip 8.0 (feature level 202). Before this feature level,
        linkifiers were always processed in order by ID, which meant users would
        need to delete and recreate them to reorder the list of linkifiers.
```